src/App.tsx

```diff
@@ -1,14 +1,77 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoModal, setTodoModal] = useState<Todo | null>(null);
+  const [selectCompleted, setSelectCompleted] = useState('all');
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    const loadGoods = async () => {
+      const goodsData = await getTodos();
+
+      setTodos(goodsData);
+    };
+
+    loadGoods();
+  }, [todos]);
+
+  const openModal = useCallback((todo: Todo) => {
```

Comment:
look at checklist pt 3 about naming your functions
```suggestion
  const handleOpenModal = useCallback((todo: Todo) => {
```

Reply:
and apply it to rest of the functions 😄